borg/testsuite/archiver.py

```diff
@@ -71,6 +71,29 @@ def __exit__(self, *args, **kw):
                 os.environ[k] = v
 
 
+def exec_cmd(*args, archiver=None, fork=False, **kw):
```

Comment:
because I didn't want to refactor all the callers. it also would add more code to the callers (see the remaining method, that returncode stuff and the assert).